System-Architecture/CSC-Overview/Auxiliary-Telescope-CSCs/index.rst

```diff
@@ -15,46 +15,45 @@ As mentioned in the :ref:`Control-Packages-General-CSC-Overview`, it is convenie
 All CSCs associated with the AuxTel receive an ``AT`` prefix to their name, such as the ATHexapod or ATDome.
 CSCs in this group are strictly associated with the operation and calibration of the AuxTel and cannot be moved between systems.
 
-Although a :ref:`List_of_AT_CSCs` is found at the bottom of this page, it is often useful to associate the CSCs by the control-package it is associated with.
+Although a full list of AuxTel CSCs, indicated by the ``AT`` prefix, are found in the `Master CSC Table <https://ts-xml.lsst.io/#master-csc-table>`__, it is generally most useful to associate the CSCs by the control-package it is associated with, as it done below.
 
 Links are provided to the user guide when available. Interface information for each CSC can be found in the `ts-xml guide <https://ts-xml.lsst.io/>`__.
 
 
 Auxiliary Telescope Control System (ATCS)
 =========================================
 
-The :ref:`ATCS class<Control-Packages-AuxTel-ATCS>` is a high level control package that provides the most user-friendly interface to the Auxiliary Telescope related CSCs related to operating the telescope and dome.
+The :ref:`ATCS class <Control-Packages-AuxTel-ATCS>` is a high level control package that provides the most user-friendly interface to the Auxiliary Telescope related CSCs related to operating the telescope and dome.
```

Comment:
Probably good to drop one of the "related"

> related CSCs related to operating

Also, may want to avoid 

> the most user-friendly

how about:

> The ATCS class is a high level control package that provides a user-friendly interface to operating the telescope and dome related CSCs on the Auxiliary Telescope.